platform/linux-generic/include/odp_packet_internal.h

```diff
@@ -248,6 +251,106 @@ static inline void packet_init(odp_packet_hdr_t *pkt_hdr, uint32_t len)
 	pkt_hdr->input = ODP_PKTIO_INVALID;
 }
 
+/**
+ * Check if copying packet metadata between pools is possible
+ *
+ * @param      src_pool     Source packet pool handle
+ * @param      dst_pool     Destination packet pool handle
+ *
+ * @retval  0 when possible without user area copy
+ * @retval >0 when possible with user area copy
+ * @retval <0 when not possible
+ */
+static inline int _odp_packet_copy_md_possible(odp_pool_t src_pool,
+					       odp_pool_t dst_pool)
+{
+	const pool_t *src_hdr;
+	const pool_t *dst_hdr;
+
+	if (src_pool == dst_pool)
+		return 0;
+
+	src_hdr = pool_entry_from_hdl(src_pool);
+	dst_hdr = pool_entry_from_hdl(dst_pool);
+
+	if (dst_hdr->param_uarea_size < src_hdr->param_uarea_size)
+		return -1;
+
+	return 1;
+}
+
+/**
```

Comment:
Doxygen style comments not needed as this is internal function (and not seen/checked by Doxygen)

Reply:
Fixed in V6.